src/core/openthread.cpp

```diff
@@ -545,6 +547,12 @@ ThreadError otRemoveUnicastAddress(otNetifAddress *address)
     return sThreadNetif->RemoveUnicastAddress(*static_cast<Ip6::NetifUnicastAddress *>(address));
 }
 
+void otSetStateChangedCallback(void *aContext, otStateChangedCallback aCallback)
```

Comment:
I suppose this nit is really a matter of personal preference, so feel free to ignore: I would personally reverse the order of the arguments. I see more APIs for callbacks take the function pointer first and then the context.

Again, this isn't a big deal, if you like it this way I'm fine with it as long as the convention is consistent across the public interface.